components/style/gecko/wrapper.rs

```diff
@@ -125,6 +137,13 @@ impl<'ln> fmt::Debug for GeckoNode<'ln> {
 
 impl<'ln> GeckoNode<'ln> {
     #[inline]
+    fn is_document(&self) -> bool {
+        // This is a DOM constant that isn't going to change.
+        const DOCUMENT_NODE: u16 = 9;
```

Comment:
Where did this number come from? I'm also inclined to put this under `gecko_bindings/mod.rs` since this looks like it's gecko-specific and more of a configuration type of variable.

Reply:
We do the same for text nodes a few lines up, and it's definitely not a Gecko-specific thing:

http://searchfox.org/mozilla-central/rev/dd47bee6468de7e1221b4d006342ad6b9813d0e5/servo/components/script/dom/webidls/Node.webidl#19

Reply:
(Also, this is a Gecko-specific file already)